components/service/glean/src/main/java/mozilla/components/service/glean/private/LabeledMetricType.kt

```diff
@@ -55,7 +55,7 @@ data class LabeledMetricType<T>(
         //   this.is-not-fine
         //   this.$isnotfine
 
-        private const val MAX_LABEL_LENGTH = 30
+        private const val MAX_LABEL_LENGTH = 61
```

Comment:
Can you file a bug for glean-core or fix it there too?

Reply:
https://github.com/mozilla/glean/pull/126